pkg/cloud/azure/actuators/machine/actuator.go

```diff
@@ -226,6 +209,34 @@ func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machi
 	return nil
 }
 
+func (a *Actuator) checkControlPlaneMachines(scope *actuators.MachineScope, cluster *clusterv1.Cluster, machine *clusterv1.Machine) (string, error) {
```

Comment:
Let's change all `%q`'s to `%s`

Reply:
Would keep this reserved for next PR, there are just too many places in the code using %q for mysterious reasons. I have incorporated the files which was touched in this PR